ios/Example/Tests/TrifleTests.swift

```diff
@@ -94,10 +108,30 @@ Rz2iDA1898Yx5ELtlQcl7QUGXUmadwzW7rpxQB5wIk46tPTEJCFmUIYwrCB\
         let otherRootCertificate = Certificate(version: 0, certificate: otherRootCertEncoded)
         let isVerified = deviceCertificate.verify(
             certificateRequest: mobileCertReq,
-            certificateChain: [],
+            intermediateChain: [],
             rootCertificate: otherRootCertificate
         )
         
         XCTAssertFalse(isVerified)
     }
+    
+    func testSignInvalidInput() throws {
```

Comment:
Only testing simple failure cases.
Success case can't be done in unit test until we either (1) mock, or (2) write proper integration tests, or (3) mock server calls via say openssl. 
For now, we test the old fashion way.